plugins/task/privacyconsent/services/provider.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Task.PrivacyConsent
```

Comment:
Most common to use lowercase letters here.
```suggestion
 * @subpackage  Task.privacyconsent
```